src/Pororoca.Desktop/Views/WebSocketConnectionView.xaml

```diff
@@ -532,19 +532,66 @@
       <Grid
           Grid.Column="2"
           Grid.Row="0"
-          RowDefinitions="Auto, * , Auto">
+          RowDefinitions="Auto, Auto, Auto, * , Auto">
+            
+          <TextBlock
+              Name="tbResTitle"
+              Grid.Column="0"
+              Grid.Row="0"
+              Margin="12,28,12,0"
+              Classes="SectionTitle"
+              Text="{Binding ResponseStatusCodeElapsedTimeTitle}"/>
+
+        <DataGrid
```

Comment:
I was thinking of a TabControl / TabItem view here, with one TabItem for exchanged messages, and other TabItem for status code and headers